.travis.yml

```diff
@@ -1,77 +0,0 @@
-language: rust
```

Comment:
I think we'll want to run our tests on *both* travis and github actions for a little while, to make sure everything is configured correctly. Could you remove this commit please, so that the travis config is still in the repo?

Reply:
Done. I'll keep it.